.github/workflows/pythonpublish.yml

```diff
@@ -14,15 +14,10 @@ jobs:
           token: "${{ secrets.PERSONAL_ACCESS_TOKEN }}"
           fetch-depth: 0
       - name: Set up Python
-        uses: actions/setup-python@v6
+        uses: astral-sh/setup-uv@v6
         with:
           python-version: "3.x"
-      - name: Install dependencies
-        run: |
-          python -m pip install -U pip poetry
-          poetry --version
+      - name: Build
+        run: uv build
       - name: Publish
-        env:
-          POETRY_HTTP_BASIC_PYPI_USERNAME: ${{ secrets.PYPI_USERNAME }}
-          POETRY_HTTP_BASIC_PYPI_PASSWORD: ${{ secrets.PYPI_PASSWORD }}
-        run: poetry publish --build
+        run: uv publish
```

Comment:
How does it load the secret?

Reply:
I've added the trusted publisher for this file, we are good to go 👍🏻